ferrocene/doc/user-manual/src/overview.rst

```diff
@@ -6,11 +6,7 @@ Overview
 
 This Manual describes the use of Ferrocene |ferrocene_version|, the |iso_ref|
 and |iec_ref| qualified version of the Rust toolchain.
-
-Ferrocene is based on:
-
-* rustc version |rust_version|.
-* rustfmt version |rustfmt_version|.
+Ferrocene is based on ``rustc`` version |rust_version|.
```

Comment:
The rustfmt version is different than the rustc version.

Reply:
the versions for each are specified [in SM](https://public-docs.ferrocene.dev/main/safety-manual/scope.html), so rustfmt does not need to be mentioned here

Reply:
What harm does it do to mention it here?

Reply:
it is another place to remember to mention tools that are qualified (which means it might get stale), when one already exists

the inital reason I decided to remove this is I was about to add `ferrocene-self-test`, and realized that the line "Ferrocene is based on" is not even right, and I saw that the tools are mentioned elsewhere anyways